### PR TITLE
Feat: extend @ORD.Extensions

### PR DESCRIPTION
**Pull request by zongqichen**
This pr is to fix the [issue#64](https://github.com/cap-js/ord/issues/64), implementing generic logic.

### CODE REVIEW

**Comment by Fannon** [comment]
@zongqichen - can you provide a short issue description (only roughly) what this PR is doing and link the related PR that this builds upon? Thanks!

**Comment by swennemers** [comment]
follow up to 63

File: lib/templates.js [hunk 1]
```diff
@@ -173,45 +177,38 @@ const createAPIResourceTemplate = (serviceName, serviceDefinition, appConfig, pa
...
-        ordId: `${appConfig.ordNamespace}:eventResource:${eventName}:v1`,
+        ordId: `${appConfig.ordNamespace}:eventResource:${serviceName}:v1`,
         title:
-            ordExtensions.title ??
             eventDefinition["@title"] ??
             eventDefinition["@Common.Label"] ??
             `ODM ${appConfig.appName.replace(/[^a-zA-Z0-9]/g, "")} Events`,
-        shortDescription: ordExtensions.shortDescription ?? "Example ODM Event",
-        description: ordExtensions.description ??
+        shortDescription: "Example ODM Event",
```
[the remark this answers is not in the document]
Comment (Fannon) [reply lib/templates.js:188]: Or maybe better:

```suggestion
        shortDescription: `${serviceName} event resource`,
```

File: lib/templates.js [hunk 2]
```diff
@@ -173,45 +177,38 @@ const createAPIResourceTemplate = (serviceName, serviceDefinition, appConfig, pa
...
+        description:
             eventDefinition['@description'] ?? eventDefinition['@Core.Description'] ??
             "CAP Event resource describing events / messages.",
-        version: ordExtensions.version ?? "1.0.0",
-        releaseStatus: ordExtensions.releaseStatus ?? "beta",
+        version: "1.0.0",
+        releaseStatus: "beta",
```
Comment (Fannon) [inline lib/templates.js:193]: Not sure if "beta" is a good default here. I would go with active / stable:

```suggestion
        releaseStatus: "active",
```

**Comment by swennemers** [comment]
@zongqichen please update the documentation in https://github.com/cap-js/ord/blob/main/ord.md

**Review by swennemers — APPROVED** [review]

**Pull request merged**